src/components/parameters/component-parameters-implementation.adb

```diff
@@ -364,6 +380,66 @@ package body Component.Parameters.Implementation is
             Set_Status (Self.Stage_Parameter (Param_Entry => Param_Entry, Value => Value));
```

Comment:
If Set_Status is only being called once in each of these functions, It might be cleaner to just inline the logic instead of having the separate Set_Status procedure.

Reply:
Cleaned up in [1e5756b](https://github.com/lasp/adamant/pull/65/commits/1e5756ba3fca1eb6e6c58154922a7be37504f8c3).